File: app/src/main/java/cz/kutner/comicsdb/connector/service/ClassifiedService.java
package cz.kutner.comicsdb.connector.service;

import java.util.List;

import cz.kutner.comicsdb.model.Classified;
import retrofit.http.GET;
import retrofit.http.Query;
import rx.Observable;

public interface ClassifiedService {
    @GET("/bazar.php")
    Observable<List<Classified>> filteredClassifiedList(@Query("str") int page, @Query("id") int categoryId, @Query("val")  String searchText);
}

File: app/src/main/kotlin/cz/kutner/comicsdb/connector/service/ClassifiedService.kt
package cz.kutner.comicsdb.connector.service

import cz.kutner.comicsdb.model.Classified
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ClassifiedService {
    @GET("/api.php?get=bazar")
    fun filteredClassifiedList(@Query("start") start: Int, @Query("records") records: Int, @Query("id") categoryId: Int): Call<List<Classified>>
}
